book.rb

```diff
@@ -0,0 +1,17 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    rental = Rental.new(date, self, person)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] Instantiating a rental object already adds the rental to the `rentals` array in the Rental class `(book.rentals.push(self))` so we don't need to add to the array here again. Kindly refactor this method to this.

  ```
  def add_rental(date, person)
    Rental.new(date, self, person)
  end
  ```

Reply:
add_rental object has been modified. Thanks for the feedback.